rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -47,6 +47,11 @@ inline QString qsv(std::string_view sv) { return QString(sv.data()); }
 
 extern const std::map<std::string_view, int> g_prx_list;
 
+namespace utils
+{
+	u32 get_thread_count();
```

Comment:
why exactly do we need all these forward declarations?

Reply:
Better than include the whole header for a single function IMO.

Reply:
If we went with this as the standard we'd have tons of forward declarations all over. Most included headers are required for only one or two functions especially when it comes to util. Slippery slope argument and all that.
Also what happens when someone needs another function? Start modifying unrelated code? This just seems like bad design to me.